template/web/wp-content/themes/<%= slug %>/app/Repositories/PostRepository.php

```diff
@@ -7,10 +7,24 @@
 
 namespace Studiometa\Repositories;
 
+use Timber\PostQuery;
+
 /** Class */
-class PostTypeRepository extends Repository {
+class PostRepository extends Repository {
+	const CLASS_TYPE = '\Timber\Post';
 	const POST_TYPES = array( 'post' ); // Main post types.
 
+	/**
+	 * Implement do_query
+	 *
+	 * @param array  $params for the query.
+	 * @param string $class_type to return.
+	 * @return \Timber\PostQuery
+	 */
+	public function do_query( $params, $class_type ) {
```

Comment:
`$class_type` parameter wasn't used.

Reply:
The `$class_type` parameter is still present, I think it can be removed.